hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/ECBlockOutputStreamEntryPool.java

```diff
@@ -55,6 +55,9 @@ public ECBlockOutputStreamEntryPool(OzoneClientConfig config,
         isMultipart, info, unsafeByteBufferConversion, xceiverClientFactory,
         openID);
     assert replicationConfig instanceof ECReplicationConfig;
+    getExcludeList()
```

Comment:
See my earlier comment, which may allow us to get rid of the background thread.

However if we need to keep the thread, I think the responsibility for starting the thread should be internal to the ExcludeList class, rather than having to ensure the user of the class needs to know about it, and start it. Should we not make ExcludeList start its own thread via its constructor or on first use?

Reply:
Same as above reply.